Automattic/wp-calypso
# Remove the theme/showcase-revamp flag

#### Changes proposed in this Pull Request

* Remove the theme/showcase-revamp flag

#### Testing instructions



* Visit the theme showcase and see that everything still works as normal

![2021-07-29_16-19](https://user-images.githubusercontent.com/937354/127567300-b72e7fe1-1485-4557-a745-5994de06ef14.png)




Related to #
